react-toggle-switch/my-react-app/src/App.tsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import './App.css';
+import './ToggleSwitch.css';
+
+function App() {
+  const [toggled, setToggle] = useState(false);
+  return (
+    <>
+      <button
```

Comment:
Looks good! just move this to the ToggleSwitch.tsx file and add a "ON" "OFF" label.